src/App.jsx

```diff
@@ -83,6 +85,11 @@ function App({
           path="/"
         />
 
+        <Route
```

Comment:
Since the setting route does not need adb at all, we can move it one routing hirarchy higer into the ```AdbRouter.jsx``` - logic wise it is very close to the About page, there we also don't need the "Connect" button